src/parameters/create_parameters.jl

```diff
@@ -36,10 +26,12 @@ function create_parameter_set(config::AtmosConfig)
         create_parameter_struct(TD.Parameters.ThermodynamicsParameters)
     rrtmgp_params = create_parameter_struct(RP.RRTMGPParameters)
     insolation_params = create_parameter_struct(IP.InsolationParameters)
-    microphys_params = create_parameter_struct(
-        CM.Parameters.CloudMicrophysicsParameters;
-        subparam_structs = (; thermo_params),
-    )
+
+    toml_dict["τ_precip"]["value"] = FT(CA.time_to_seconds(parsed_args["dt"]))
```

Comment:
Probably a temporary solution? The alternative is for me to make the CloudMicrophysics structs be `kwdef`

Reply:
As we discussed, using `@kwdef` does not seem logical when default values are not being set! Updating the value for the dict key looks like the more natural thing to do!